### PR TITLE
[Feature] Update pages to mock API calls + create page structure

### PR DESCRIPTION
**Pull request by krishan711**
domains page:
Screen Shot 2020-12-04 at 10 54 10

domain page:
Screen Shot 2020-12-04 at 10 54 16

### CODE REVIEW

File: app/src/pages/domainsPage/domainsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  React.useEffect((): void => {
+    loadHomeDomains();
+  }, []);
+
+  const asyncSleep = (sleepTime: number): Promise<void> => {
+    return new Promise(resolve => setTimeout(resolve, sleepTime));
+  }
+
+  const loadHomeDomains = (): void => {
+    asyncSleep(300).then((): void => {
```
Comment (rikhilrai) [inline app/src/pages/domainsPage/domainsPage.tsx:22]: why sleep here?

File: app/src/pages/domainsPage/components/linkRow.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+interface ILinkRowProps {
+  link: Link;
+}
+
+export const LinkRow = (props: ILinkRowProps): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Horizontal} shouldAddGutters={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Start} isFullWidth={false}>
+      <Text>{`/${props.link.sourcePath}`}</Text>
+      <KibaIcon iconId='mui-arrow-right-alt' />
+      <Text>{`${props.link.destination}`}</Text>
```
Comment (rikhilrai) [inline app/src/pages/domainsPage/components/linkRow.tsx]: i don't think interpolation is required here. did you keep it like this to be consistent with line 13?

File: app/src/pages/domainsPage/components/homeDomainCard.tsx [hunk 3]
```diff
@@ -30,6 +17,7 @@ export const HomeDomainCard = (props: IHomeDomainCardProps): React.ReactElement
         <Stack direction={Direction.Horizontal} shouldAddGutters={true} childAlignment={Alignment.Center}>
           <Text variant='header3'>{props.homeDomain.domain.url}</Text>
           <Text variant='note'>{props.homeDomain.domain.isVerified ? 'verified' : 'unverified'}</Text>
+          {/* <Pill variant={props.homeDomain.domain.isVerified ? 'success-small' : 'default-small'} text={props.homeDomain.domain.isVerified ? 'verified' : 'unverified'} /> */}
```
Comment (rikhilrai) [inline app/src/pages/domainsPage/components/homeDomainCard.tsx]: commented out?

File: app/src/pages/domainPage/domainPage.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  domainId: string;
+}
+export const DomainPage = (props: IDomainPageProps): React.ReactElement => {
+  const [domain, setDomain] = React.useState<Domain | null>(null);
+
+  React.useEffect((): void => {
+    loadDomain();
+  }, [props.domainId]);
+
+  const asyncSleep = (sleepTime: number): Promise<void> => {
```
Comment (rikhilrai) [inline app/src/pages/domainPage/domainPage.tsx]: duplicated, move to util?

**Review by rikhilrai — APPROVED** [review]

File: app/src/pages/domainsPage/domainsPage.tsx (hunk 1 above)
Comment (krishan711) [reply app/src/pages/domainsPage/domainsPage.tsx:22]: just to mock api delay

File: app/src/pages/domainsPage/components/homeDomainCard.tsx (hunk 3 above)
Comment (krishan711) [reply app/src/pages/domainsPage/components/homeDomainCard.tsx]: the pill is huge at the moment, looks like this 🤦‍♂️
Screen Shot 2020-12-04 at 10 45 33

Comment (krishan711) [reply app/src/pages/domainsPage/components/homeDomainCard.tsx]: oh actually i can update the styles here, doesnt need to be in main library, will do now

Comment (krishan711) [reply app/src/pages/domainsPage/components/homeDomainCard.tsx]: updated, silly me 🤦‍♂️

**Review by rikhilrai — APPROVED** [review]

**Pull request merged**